sdk/tables/azure-data-tables/tests/test_table.py

```diff
@@ -108,7 +111,9 @@ def test_create_properties(self, resource_group, location, storage_account, stor
         ps = ts.get_service_properties()
         ts.delete_table(table_name)
 
-    @GlobalStorageAccountPreparer()
+    # @pytest.mark.skip("pending")
```

Comment:
Are these skips needed?  deprecated?  is the skip on line 90 needed?

Reply:
No these are not needed, I keep the commented out ones to make it easier on myself when debugging just a few tests.